src/webui-embed.html

```diff
@@ -93,16 +95,40 @@
                 }
             }
 
-            function format_hashrate(hr)
+            function format_hashes(h)
             {
-                if (hr < 1000)
-                    return parseInt(hr) + " H/s";
-                else if (hr < 1000000)
-                    return parseFloat(hr/1000).toFixed(2) + " KH/s";
-                else if (hr < 1000000000)
-                    return parseFloat(hr/1000000).toFixed(2) + " MH/s";
+                if (h < 1e-9)
+                    return parseFloat(h*1e+12).toFixed(2) + " pH";
```

Comment:
Zero now displays as `0.00 pH`. where it should just be `0 H`.

Reply:
I'll treat values within [0, 1e-12) separately as '0 H'

Reply:
why not if h == 0 return "0 H' followed by else if (h < 1e-9)

Reply:
This way not only absolute zero will be displayed as '0 H'.
I suppose there is no reason to have '0 H' for 0 and '0.00 pH' for 1e-13